### PR TITLE
[DOCS] Fix typo in docs

### PR DESCRIPTION
**Pull request by uncleGen**
## What changes were proposed in this pull request?

Fix typo in docs

### CODE REVIEW

**Comment by uncleGen** [comment]
cc @srowen

**Comment by srowen** [comment]
To cut down on the number of these PRs, please spend some time to spell check the other docs too.

**Comment by uncleGen** [comment]
@srowen I have checked on one sweep of all the docs.

File: docs/index.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ It also supports a rich set of higher-level tools including [Spark SQL](sql-prog
 Get Spark from the [downloads page](http://spark.apache.org/downloads.html) of the project website. This documentation is for Spark version {{site.SPARK_VERSION}}. Spark uses Hadoop's client libraries for HDFS and YARN. Downloads are pre-packaged for a handful of popular Hadoop versions.
 Users can also download a "Hadoop free" binary and run Spark with any Hadoop version
 [by augmenting Spark's classpath](hadoop-provided.html).
-Scala and Java users can include Spark in their projects using its maven cooridnates and in the future Python users can also install Spark from PyPI.
+Scala and Java users can include Spark in their projects using its maven coordinates and in the future Python users can also install Spark from PyPI.
```
Comment (srowen) [inline docs/index.md]: Even here, maven -> Maven. That's the kind of other thing you can fix while you're at it, please.

Comment (uncleGen) [inline docs/index.md]: ok

**Review by srowen — APPROVED** [review]

**Comment by srowen** [comment]
Merged to master

**Pull request closed without merging**